### PR TITLE
YEE

### PR DESCRIPTION
**Pull request by Mnpn**
SUCH DANK MUCH WOW

### CODE REVIEW

File: Dank/images.json [hunk 1]
```diff
@@ -25,4 +25,5 @@
 	"this is fine": "http://i.imgur.com/c4jt321.png",
 	"i like trains": "http://vignette2.wikia.nocookie.net/eddsfanon/images/1/11/710947_1304793662055_full.png/revision/latest?cb=20130427092052",
 	"poop": "https://cdn.discordapp.com/attachments/228221346608316417/296671987357581313/trumppoop.jpg"
```
Comment (jD91mZM2) [inline Dank/images.json]: I TINK U MISD COMMA? SRY I CANT MERGE DISS

**Review by jD91mZM2 — CHANGES_REQUESTED** [review]
pls

File: Dank/images.json (hunk 1 above)
Comment (Mnpn) [reply Dank/images.json]: WAT NO I HAD COMMA IS PESSIMIST!=!

**Comment by Mnpn** [comment]
i fix @LEGOlord208

**Review by jD91mZM2 — APPROVED** [review]

**Comment by jD91mZM2** [comment]
U B GOOD WORK. I B PROUD OF YU SON

**Pull request merged**